datacenter_describe.go

```diff
@@ -0,0 +1,61 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func newDatacenterDescribeCommand(cli *CLI) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "describe [FLAGS] DATACENTER",
+		Short:                 "Describe a datacenter",
+		Args:                  cobra.ExactArgs(1),
+		TraverseChildren:      true,
+		DisableFlagsInUseLine: true,
+		RunE: cli.wrap(runDatacenterDescribe),
+	}
+	return cmd
+}
+
+func runDatacenterDescribe(cli *CLI, cmd *cobra.Command, args []string) error {
+	idOrName := args[0]
+	datacenter, _, err := cli.Client().Datacenter.Get(cli.Context, idOrName)
+	if err != nil {
+		return err
+	}
+	if datacenter == nil {
+		return fmt.Errorf("datacenter not found: %s", idOrName)
+	}
+
+	fmt.Printf("ID:\t\t%d\n", datacenter.ID)
+	fmt.Printf("Name:\t\t%s\n", datacenter.Name)
+	fmt.Printf("Description:\t%s\n", datacenter.Description)
+	fmt.Printf("Location:\n")
+	fmt.Printf("  Name:\t\t%s\n", datacenter.Location.Name)
+	fmt.Printf("  Description:\t%s\n", datacenter.Location.Description)
+	fmt.Printf("  Country:\t%s\n", datacenter.Location.Country)
+	fmt.Printf("  City:\t\t%s\n", datacenter.Location.City)
+	fmt.Printf("  Latitude:\t%f\n", datacenter.Location.Latitude)
+	fmt.Printf("  Longitude:\t%f\n", datacenter.Location.Longitude)
+	fmt.Printf("ServerTypes:\n")
+	st := datacenter.ServerTypes
+	fmt.Printf("  Available:\n")
+	if len(st.Available) > 0 {
+		for _, t := range st.Available {
+			fmt.Printf("  - ID:\t%d\n", t.ID)
```

Comment:
To me, showing just the ID is useless. We should fetch the server types and, at least, show the name.